src/pages/Admin/Posts/UserPostsManagementScreen.css

```
@@ -1,305 +1,731 @@
+.posts-detail-row>span:first-child {
+  min-width: 108px;
+  flex: 0 0 108px;
+  color: var(--neutral-500);
+}
+
+.posts-detail-row>strong {
+  color: var(--neutral-900);
+  flex: 1;
+  min-width: 0;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)